Renamed appearance tab to membrane tab.

buckly90 opened the pull request:
closes #1869

hhyyrylainen commented:
Need to run localization script before this can be merged.

hhyyrylainen commented:
Also it seems that the localizations don't have plain MEMBRANE key in them, so a "translation" for MEMBRANE to "Membrane" needs to be added to en.po

hhyyrylainen commented on locale/en.po:
```diff
@@ -1718,176 +1718,177 @@ msgstr "Auto-evo failed to run"
-msgid "APPEARANCE"
-msgstr "Appearance"
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1603
+#, fuzzy
```
This line should be removed because this is a proper translation, right?
Or should this translation actually be "Membrane"

hhyyrylainen replied:
I just tested and it looks like Godot ignores translations marked as fuzzy entirely. So the tab actually now ends up being "MEMBRANE" which is not consistent with the text to its left or right.

FuJa0815 reviewed (commented):
There are still merge conflicts. I can try to resolve them if you want me to.

buckly90 commented:
> 
> 
> There are still merge conflicts. I can try to resolve them if you want me to.

Sorry about the late reply, I had forgotten about this.
I would like to at least try to resolve the conflicts myself first as I don't like shirking work I started at the expense of others like that.

Any tips on how to go about this? The conflicting files just seem to be the files I changed, which I assume will overwrite any changes others have made in the event that my branch is outdated (It most likely is). My usual method of updating branches of mine is to just take out the files I changed, pull from master, and then replace them. But that's normally when I just change one or two.

FuJa0815 commented:
I would do the following:
1. `git pull`
2. `git merge master`
3. resolve the conflicts using any editor. I use Visual Studio Code

I resolve language files like that:
I go through one file manually and check if the current change is applicable everywhere. Line numbers don't really matter.
If that is the case I accept all current changes (VSC: Ctrl+Shift+P => Merge Conflict: Accept all Current) in all language files.
After resolving every conflict I rerun `update_localization.ru`. There will probably be some duplicate definitions you will have to resolve manually by removing one of them.

I resolve the scene file the following way:
Margin conflicts don't really matter. Choose whatever seems better.
Every other conflict requires case to case care.

After everything is done: `git commit -am "resolved merge conflicts with master" && git push`

FuJa0815 commented on src/microbe_stage/editor/MicrobeEditor.tscn:
```diff
@@ -1562,8 +1562,13 @@ custom_colors/font_color_pressed = Color( 0, 0, 0, 1 )
 toggle_mode = true
 action_mode = 0
 enabled_focus_mode = 0
+<<<<<<< HEAD
+group = SubResource( 21 )
+text = "MEMBRANE"
+=======
 group = SubResource( 15 )
 text = "APPEARANCE"
+>>>>>>> master
```
```suggestion
group = SubResource( 15 )
text = "MEMBRANE"
```

buckly90 replied:
Strange, my editor wouldn't point that one out to me at all, so I had to find it manually.

FuJa0815 replied:
It isn't fixed, there is still one bad line (`>>>>>>> master`). Fix the file via a text editor.

buckly90 replied:
Should be gone now, apologies for the time.

FuJa0815 reviewed (changes requested):
There is a merge conflict present.
ID 15 should work, because the other tab buttons also use group 15 (see line 1542)

hhyyrylainen commented:
Looks like this only has part of the required changes, that isn't consistent between the translation files.

Kasterisk reviewed (approved):
Looks to be correct codewise and ingame, I'd say this is now good to merge 👍
